File: src/backend/ApiManager.Core/Entities/Abstractions/EntityBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ApiManager.Core.Entities.Abstractions
{
    public class EntityBase<TKey> : IEntityBase<TKey>
    {
        public TKey? Id { get; set; }
        public bool IsDeleted { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime LatestUpdatedDate { get; set; }
        public string? CreatedAccountId { get; set; }
        public string? LatestUpdatedAccountId { get; set; }
    }
}

File: src/backend_node/api_manager/src/core/entities/entityBase.ts
import { Column, PrimaryColumn } from 'typeorm'

export class EntityBase {

    @PrimaryColumn({
        name:'id'
    })
    public id: string

    @Column({
        name:'is_deleted'
    })
    public isDeleted: boolean

    @Column({
        name:'created_date'
    })
    public createdDate: Date

    @Column({
        name:'latest_updated_date'
    })
    public latestUpdatedDate: Date

    @Column({
        name:'created_account_id'
    })
    public createdAccountId: string

    @Column({
        name:'latest_updated_account_id'
    })
    public latestUpdatedAccountId: string
}